src/core/algorithms/fd/sfd/cords.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "algorithm.h"
+#include "algorithms/fd/fd_algorithm.h"
+#include "config/equal_nulls/type.h"
+#include "config/tabular_data/input_table_type.h"
+#include "correlation.h"
+#include "frequency_handler.h"
+#include "model/table/column.h"
+#include "model/table/column_index.h"
+#include "model/table/column_layout_typed_relation_data.h"
+#include "sample.h"
+
+namespace algos {
+/*  */
+class Cords : public FDAlgorithm {
```

Comment:
Yes, we discussed that Cords should be an inheritor of FDAlgorithm, since SFD is a relaxation of FD and there are no other (known to us) SFD mining algorithms so making some kind of intermediate class is redundant.